QuickOSM/test/test_query_factory.py

```diff
@@ -175,8 +177,29 @@ def test_query(q, xml, xml_with_template, human_label):
             '<print mode="body"/>'
             '</osm-script>'
         )
+        expected_oql = (
+            '[out:xml] [timeout:25];\n'
+            '(\n'
+            '    node( bbox="custom");\n'
+            '    way( bbox="custom");\n'
+            '    relation( bbox="custom");\n'
+            ');\n'
+            '(._;>;);\n'
+            'out body;'
+        )
+        expected_oql_with_template = (
+            '[out:xml] [timeout:25];'
+            '('
+            'node( {{bbox}});'
+            'way( {{bbox}});'
+            'relation( {{bbox}});'
+            ');'
+            '(._;>;);'
+            'out body;'
+        )
         human = 'All OSM objects in the canvas or layer extent are going to be downloaded.'
-        test_query(query, expected_xml, expected_xml_with_template, human)
+        test_query(query, expected_xml, expected_xml_with_template,
+                   expected_oql, expected_oql_with_template, human)
```

Comment:
```suggestion
                   expected_oql, expected_oql_with_template, human)
```

For next time, keep pulling left. After `test_query(`, press enter. So you will have **always** a multiple of 4 spaces for the indentation.
The comment for other lines in the file below.
You can skip this one now.